app/models/generic_file.rb

```diff
@@ -1,15 +1,15 @@
 class GenericFile < ActiveFedora::Base
   include Sufia::GenericFile
 
-  property :production_id, predicate: ::RDF::URI('http://docs.osfashland.org/terms/production_id'), multiple: false
+  property :production_ids, predicate: ::RDF::URI('http://docs.osfashland.org/terms/production_ids'), multiple: true
```

Comment:
Line is too long. [116/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.